src/pages/Authentication/Signup/Signup.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { FaRegEyeSlash } from "react-icons/fa";
+import { Link } from "react-router-dom";
+import { Navbar } from "../../../barrelexport/Componentutil";
```

Comment:
Just a suggestion but maybe you can do barrel export from the same folder where the components are situated. Because if someone wants to track down the component they will have to follow a little longer path. Tell me if I'm wrong.

Reply:
First , I did like that , but then I thought that all barrels should at one root place ,so i can find better.